### ISSUE TITLE
Windows only?

### ISSUE THREAD

**Issue by mimooh** [Tugcga/PyRecastDetour#8]
This looks like a python interface to a c++ recast-detour via pybind11 which is a modern alternative to a troublesome libboost, right? The provided libs are windows only, but can I compile https://github.com/Tugcga/PyRecastDetour-Sources for linux?

**Comment by Tugcga** [comment]
Yes, you are right. This is bindings for some functions of the original RecastNavigation library. You can try to compile it from sources for Linux, potentially all should works. But you need precompiled RecastNavigation for your target platform.

**Comment by p-delorme** [comment]
Hey @mimooh, I was curious to know if you ever managed to get to working on Linux?
